### PR TITLE
Remove all stress tests related to uploadFromUrl APIs

### PR DESCRIPTION
**Pull request by ibrahimrabab**
# Description

- Removing tests that test for uploadFromUrl APIs, as these tests do not necessarily test the resilience of the SDK.
- Increasing duration of large file size tests

### CODE REVIEW

File: sdk/storage/azure-storage-file-share-stress/src/main/java/com/azure/storage/file/share/stress/App.java [hunk 1]
```diff
@@ -18,8 +18,7 @@ public static void main(String[] args) {
             OpenInputStream.class,
             Upload.class,
             UploadFromFile.class,
-            UploadRange.class,
-            UploadRangeFromUrl.class
```
Comment (kyleknap) [inline sdk/storage/azure-storage-file-share-stress/src/main/java/com/azure/storage/file/share/stress/App.java:22]: Do we want to remove the `UploadRangeFromUrl.java` file as well?

**Review by kyleknap — COMMENTED** [review]
Looks good. Just had one small comment

File: sdk/storage/azure-storage-file-share-stress/src/main/java/com/azure/storage/file/share/stress/App.java (hunk 1 above)
Comment (ibrahimrabab) [reply sdk/storage/azure-storage-file-share-stress/src/main/java/com/azure/storage/file/share/stress/App.java:22]: Thank you for catching this! Removed now

**Review by kyleknap — APPROVED** [review]
Looks good :ship:

**Pull request merged**